src/RuleSet.php

```diff
@@ -56,12 +56,53 @@ final class RuleSet implements RuleSetInterface
             'switch_case_space' => true,
             'visibility_required' => true,
         ],
-        '@Symfony' => [
+        '@PSR12' => [
             '@PSR2' => true,
-            'array_syntax' => ['syntax' => 'short'],
-            'backtick_to_shell_exec' => true,
             'binary_operator_spaces' => true,
             'blank_line_after_opening_tag' => true,
+            'compact_nullable_typehint' => true,
+            'declare_equal_normalize' => true,
+            'lowercase_cast' => true,
+            'lowercase_static_reference' => true,
+            'new_with_braces' => true,
+            'no_blank_lines_after_class_opening' => true,
+            'no_leading_import_slash' => true,
+            'no_whitespace_in_blank_line' => true,
+            'ordered_class_elements' => [
+                'order' => [
+                    'use_trait',
+                ],
+            ],
+            'ordered_imports' => [
+                'imports_order' => [
+                    'class',
+                    'function',
+                    'const',
+                ],
+                'sort_algorithm' => 'none',
+            ],
+            'return_type_declaration' => true,
+            'short_scalar_cast' => true,
+            'single_blank_line_before_namespace' => true,
+            'single_trait_insert_per_statement' => true,
+            'ternary_operator_spaces' => true,
+            'unary_operator_spaces' => true,
```

Comment:
Actually, PSR-12 says nothing about the space between an `!` and an expression, so this rule is too strict.